### PR TITLE
fixed the tests

### PR DESCRIPTION
**Pull request by iantelli**
time to get pizza

### CODE REVIEW

**Comment by iliaamiri** [comment]
where?

**Comment by iantelli** [comment]
Down the street, there's a local pizzeria

File: src/_domain/injectableTokens.ts [hunk 1]
```diff
@@ -18,8 +16,10 @@ const injectableTokens = {
     IPostTypesRepository: Symbol("IPostTypesRepository"),
     IPostTagsRepository: Symbol("IPostTagsRepository"),
     IPostsService: Symbol("IPostsService"),
+    IPostAwardRepository: Symbol("IPostAwardRepository"),
```
Comment (iliaamiri) [inline src/_domain/injectableTokens.ts:19]: Amazing

File: src/posts/controllers/postTags.controller.ts [hunk 2]
```diff
@@ -19,18 +24,13 @@ import { IPostTagsRepository } from "../services/postTagRepository/postTags.repo
...
-    ) {
+    constructor(@Inject(_$.IPostTagsRepository) postTagsRepository: IPostTagsRepository) {
         this._postTagsRepository = postTagsRepository;
     }
 
     @Get()
     public async index(): Promise<PostTag[] | Error> {
         let postTags = await this._postTagsRepository.findAll();
-        for (let i = 0; i < postTags.length; i++) {
-            postTags[i] = await postTags[i];
```
Comment (iliaamiri) [inline src/posts/controllers/postTags.controller.ts:32]: Thank you

File: src/posts/posts.module.ts [hunk 3]
```diff
@@ -10,6 +10,7 @@ import { PostsRepository } from "./services/postRepository/posts.respository";
 import { PostsService } from "./services/posts.service";
 import { PostTagsRepository } from "./services/postTagRepository/postTags.repository";
 import { PostTypesRepository } from "./services/postTypeRepository/postTypes.repository";
+import { PostAwardRepository } from "./services/postAwardRepository/postAward.repository";
```
Comment (iliaamiri) [inline src/posts/posts.module.ts:13]: Nice

**Review by iliaamiri — APPROVED** [review]
LGTM 🍕

**Pull request merged**